mkosi/__init__.py

```diff
@@ -3975,20 +3975,72 @@ def install_unified_kernel(
             cmdline = state.workspace / "cmdline"
             cmdline.write_text(boot_options)
             initrd = state.root / boot_directory(state, kver) / "initrd"
+            pcrsig = None
+            pcrpkey = None
+
+            # If a SecureBoot key is configured, and we have the
+            # systemd-measure binary around, then also include a
+            # signature of expected PCR 11 values in the kernel image
+            if state.config.secure_boot:
+                if shutil.which('systemd-measure'):
+                    with complete_step("Generating PCR 11 signature…"):
+                        from cryptography import x509
+                        from cryptography.hazmat.primitives import serialization
```

Comment:
Until now `cryptography` was only imported in `make_verity_sig`, so not using verity one could forego the cryptography dependency. Since this is now in the main path we should decide whether we skip this and warn or whether we say that `cryptography` is now an mkosi dependency proper, since people usually won't get do decide whether `systemd-measure` is installed or not.